rosdep/base.yaml

```diff
@@ -823,6 +823,14 @@ geographiclib:
     vivid: [libgeographic-dev]
     wily: [libgeographic-dev]
     xenial: [libgeographic-dev]
+geographiclib-tools:
+  debian: [geographiclib-tools]
+  ubuntu:
+    precise: [geographiclib-tools]
```

Comment:
As long as the package is generally available the more compact version is better, since it doesn't need updating for every future ubuntu release. And it will work for the intermediate ones that you didn't call out either.